test_runner/fixtures/neon_fixtures.py

```diff
@@ -1990,10 +1990,12 @@ def log_contains(
                 if cur_line_no < skip_until_line_no:
                     cur_line_no += 1
                     continue
-                if contains_re.search(line):
+                elif contains_re.search(line):
                     # found it!
                     cur_line_no += 1
                     return (line, LogCursor(cur_line_no))
+                else:
+                    cur_line_no += 1
```

Comment:
Interesting. So the issue was that we returned a `LogCursor` which wasn't pointing to the right line when we eventually found the match?

Reply:
Ah yes, but that was my fault since when I copied the code I messed it up. This is the same code as on master

Reply:
Ah I see. Missed that in the review.